english/mmacoder/mmacoder.md

```diff
@@ -0,0 +1 @@
+[mmacoder.com](http://www.mmacoder.com) is where I will be blogging about my experience with the Udacity Nanodegree Plus program. Follow me on my blog and everywhere on social media to witness the rise of MMACoder to MMA and mobile app development stardom!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.